File: olo-promotions-sdk-csharp/Olo.Promotions.SDK/Responses/VoidAccrualResponse.cs
using Olo.Promotions.SDK.Responses.Models;

namespace Olo.Promotions.SDK.Responses
{
    /// <summary>
    /// Returned for a successful call to the Void Accrual endpoint.
    /// </summary>
    public class VoidAccrualResponse
    {
        public Transaction Transaction { get; set; }
    }
}

File: olo-promotions-sdk-go/pkg/promotions/models/responses/accruals/void_accrual_response.go
package accruals

import "olo-promotions-sdk-go/pkg/promotions/models/responses"

// VoidAccrualResponse is returned for a successful call to the Void Accrual endpoint.
type VoidAccrualResponse struct {
	// Transaction contains the transaction details for the voided accrual.
	Transaction responses.Transaction `json:"transaction"`
}
